.github/renovate.json5

```diff
@@ -0,0 +1,42 @@
+{
+    "$schema": "https://docs.renovatebot.com/renovate-schema.json",
+    "extends": [
+        "config:recommended",
+        ":approveMajorUpdates",
+        ":semanticCommitScopeDisabled"
+    ],
+    "ignorePresets": [":semanticPrefixFixDepsChoreOthers"],
+    "internalChecksFilter": "strict",
```

Comment:
```suggestion
```

By default, Renovate uses `internalChecksFilter=strict` [^1]. So you can drop the explicit configuration here, if you want.

[^1]: [Renovate docs, `internalChecksFilter` config option ](https://docs.renovatebot.com/configuration-options/#internalchecksfilter)